cicerone/spec/response.py

```diff
@@ -0,0 +1,30 @@
+"""Response model for OpenAPI responses.
+
+References:
+- OpenAPI 3.x Response Object: https://spec.openapis.org/oas/v3.1.0#response-object
+- Swagger 2.0 Response Object: https://swagger.io/specification/v2/#response-object
+"""
+
+from typing import Any
+
+from pydantic import BaseModel, Field
+
+
+class Response(BaseModel):
+    """Represents an OpenAPI response object."""
+
+    # Allow extra fields to support vendor extensions and future spec additions
+    model_config = {"extra": "allow"}
+
+    description: str | None = None
+    content: dict[str, Any] = Field(default_factory=dict)
```

Comment:
This is pretty lazy, check the spec for this object and you'll see these aren't `Any` and in fact are objects themselves, let us make sure all these objects exist as pydantic models and not use `Any` here unless the open api spec says it should be `any`